fgd/point/prop/prop_testchamber_door.fgd

```diff
@@ -17,6 +15,9 @@
 	open(boolean) : "Start Open" : 0 : "If set, the door starts in the open state."
 	locked(boolean) : "Start Locked" : 0 : "If set, the door starts locked and can no longer change open or closed state."
 
+	skin(integer) : "Skin" : 0 : "Skin to use in game"
+	model(studio) : "Model" : "models/props/portal_door_combined.mdl" : "Model to use in-game"
+
```

Comment:
Use SetModel base?

Reply:
I mean it works fine in-game and is inline with the other Portal PR I did.

Reply:
Nevermind I see the default model, oops!